app/mailers/custom_devise_mailer.rb

```diff
@@ -7,7 +7,9 @@ def subject_for(key)
     # Replace the invitation instruction subject for partner users
     # that were invited by other partner users.
 
-    if resource.is_a?(PartnerUser) && resource.id != resource.partner.primary_user&.id
+    if resource.is_a?(PartnerUser) && resource.id == resource.partner.primary_user&.id
+      "You've been invited to be a partner with #{resource.partner.organization.name}"
+    elsif resource.is_a?(PartnerUser) && resource.id != resource.partner.primary_user&.id
       "You've been invited to #{resource.partner.name}'s partnerbase account"
     else
       super
```

Comment:
Nit: a nested `if/else` might make it clearer that the first two cases only differ on whether `resource.id == resource.partner.primary_user&.id`

```suggestion
    if resource.is_a?(PartnerUser)
      if resource.id == resource.partner.primary_user&.id
        "You've been invited to be a partner with #{resource.partner.organization.name}"
      else
        "You've been invited to #{resource.partner.name}'s partnerbase account"
      end
    else
      super
```